package.json

```diff
@@ -250,6 +250,12 @@
           "invoice.item.pay"
         ]
       },
+      {
+        "permissionName": "ui-invoice.cancel",
+        "displayName": "Invoice: Cancel invoices",
+        "visible": true,
+        "description": ""
```

Comment:
how is it possible to use put when no permission for it?

Reply:
add subpermission for 'put'